test/graphkb.test.js

```diff
@@ -0,0 +1,175 @@
+const { simplifyRecordsLinks, shouldUpdate } = require('../src/graphkb');
+
+describe('shouldUpdate', () => {
+    describe('disease', () => {
+        const model = 'disease';
+        const originalContent = {
+            '@class': 'Disease',
+            '@rid': '#133:8',
+            alias: true,
+            createdAt: 1565314461881,
+            createdBy: '#29:0',
+            deprecated: false,
+            description: 'congenital abnormality characterized by the presence of only one kidney.',
+            displayName: 'congenital single kidney [c101220]',
+            history: '#135:28899',
+            in_AliasOf: [
+                '#67:21022',
+                '#66:23',
+            ],
+            name: 'congenital single kidney',
+            out_AliasOf: [
+                '#66:31991',
+            ],
+            source: '#40:3',
+            sourceId: 'c101220',
+            updatedAt: 1594438640025,
+            updatedBy: '#29:0',
+            url: 'http://ncicb.nci.nih.gov/xml/owl/evs/thesaurus.owl#c101220',
+            uuid: '709eb34b-27ff-42f5-be0c-9051c639deb0',
+        };
+        const excludedFields = ['displayName'];
```

Comment:
non-blocing since this is really just nitpicking, but b/c this variable is small and critical to the outcome of the test i would just pass it in the test and not define a variable at all here

Reply:
Ok, I'll follow that rule to my best in the future.